packages/edit-site/src/components/add-new-template/utils.js

```diff
@@ -408,6 +408,62 @@ export const useTaxonomiesMenuItems = ( onClickMenuItem ) => {
 	return taxonomiesMenuItems;
 };
 
+const AUTHOR_TEMPLATE_PREFIX = { user: 'author' };
```

Comment:
Why have this outside? It's very specific to the author hook.

Reply:
Because if we had it inside each time, the hook executes, we would create a new object that in turn makes the hook return a new object that is then passed to other hooks with useSelect that ends making our hook execute, and we are in an infinite loop (you can try). The alternative would be to have it inside with a useMemo, but that is a waste for a constant a defined the constant as a property of function.